site/about2.html

```diff
@@ -0,0 +1,44 @@
+
+
+<html>
+
+	<link rel="stylesheet" href="style_about.css"
```

Comment:
Make sure your ```<link>``` tag is inside of your ```<head>``` tag.  Also, don't forget to close the tag - it's missing the ending '>'.  

Most browsers are good about correcting for things like this (which is your CSS is displaying in Chrome), but some are not as good and having tags that are not closed will cause issues.